### PR TITLE
Misspells fix

### PR DESCRIPTION
**Pull request by aregm**
Fixed all misspells from goreport card.

### CODE REVIEW

**Comment by ifilippov** [comment]
Can you please merge all your changes into one commit because all of them all have one meaning

**Comment by aregm** [comment]
You can squash them all to one commit during merge.

File: test/performance/latency/latency-part1.go [hunk 1]
```diff
@@ -186,7 +186,6 @@ func latenciesLogger(ch <-chan time.Duration, stop <-chan string) {
 	for {
 		select {
 		case lat := <-ch:
-			sum += lat
```
Comment (gshimansky) [inline test/performance/latency/latency-part1.go]: Why did you remove this line?

**Comment by ifilippov** [comment]
No. We need one commit with normal message like "Correction of misspells"

File: test/performance/latency/latency-part1.go (hunk 1 above)
Comment (aregm) [reply test/performance/latency/latency-part1.go]: unused

Comment (ifilippov) [reply test/performance/latency/latency-part1.go]: What about sum := time.Duration(0) ?

Comment (aregm) [reply test/performance/latency/latency-part1.go]: Nice catch!
